<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Load</class>
 <widget class="QDialog" name="Load">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Route</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>630</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>生成路径</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>206</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>幼圆</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>起始点位置:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>G楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>十公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>主楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>五公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>操场</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>六公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>七公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>八公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学子餐厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学苑餐厅</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>幼圆</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>到达点位置:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>G楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>十公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>主楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>五公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>操场</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>六公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>七公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N楼</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>八公寓</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学子餐厅</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学苑餐厅</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>路径长度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Line_RouteLength"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>640</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>95</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="Mapshow">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>生成地图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Routeclc">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>清除路径</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
